### ISSUE TITLE
Deck stats show only half of predicted workload

### ISSUE THREAD

**Issue by leturbinator** [cjdduarte/MDS_Time_Left#9]
<img width="584" alt="Screen Shot 2020-05-14 at 10 01 16 AM" src="https://user-images.githubusercontent.com/65351722/81943835-e1f33900-95c9-11ea-8fea-f8e58d610f63.png">

As the picture shows, I have a predicted amount of 1494 cards to do. However, when I click on the actual deck, it shows that I have 2400 cards. How do I fix this so I see the actual number due? Thank you so much in advance :)

**Comment by cjdduarte** [comment]
Where are you reading the value of 2,400? Can you show that value?

**Comment by leturbinator** [comment]
<img width="741" alt="image" src="https://user-images.githubusercontent.com/65351722/81948054-4a90e480-95cf-11ea-8c67-f369c1ead9c6.png">

**Comment by leturbinator** [comment]
there's a discrepancy between what is on my home screen vs. what is actually left. I'm really not quite sure how this happened but I would appreciate any and all help.

**Comment by cjdduarte** [comment]
I will investigate here.

It would help if you installed this add-on here [[**Enhance main window**](https://ankiweb.net/shared/info/877182321)] and send me a print of your home screen again.

What version of your Anki? 2.0, 2.1.25 or other?

**Comment by leturbinator** [comment]
<img width="1651" alt="image" src="https://user-images.githubusercontent.com/65351722/81960995-c430ce80-95df-11ea-9e16-40c8211f76f9.png">

**Comment by leturbinator** [comment]
I believe im running Anki 2.1.25

**Comment by cjdduarte** [comment]
![image](https://user-images.githubusercontent.com/42243532/81979899-17b51380-9604-11ea-893a-8772b7ab3a4c.png)

I believe I found out what's going on, but for that, I need you to replace this file in the addon you just installed.

![image](https://user-images.githubusercontent.com/42243532/81980835-b7bf6c80-9605-11ea-9725-ce91219d53a6.png)

Unzip the meta.json file inside the addon folder. Restart Anki and send the new screen.
But also send the screen where the account is not the same (study screen).
[meta.zip](https://github.com/cjdduarte/MDS_Time_Left/files/4630638/meta.zip)

**Comment by cjdduarte** [comment]
I would like to see something like this:

![image](https://user-images.githubusercontent.com/42243532/81981199-5cda4500-9606-11ea-8b8a-7c54743ead5b.png)

vs

![image](https://user-images.githubusercontent.com/42243532/81981245-6b286100-9606-11ea-8dff-0ebe9a51975b.png)

**Comment by leturbinator** [comment]
Alright, so I followed what you said. I deleted the meta.json file that was in the 877182321 folder and opened the meta.zip file. Then once the meta.json file appeared again, I deleted the .zip

**Comment by leturbinator** [comment]
here's what everything looks like
<img width="1553" alt="Screen Shot 2020-05-14 at 8 08 05 PM" src="https://user-images.githubusercontent.com/65351722/81997892-0decd980-961f-11ea-90fb-c848747866fe.png">
<img width="1385" alt="Screen Shot 2020-05-14 at 8 07 54 PM" src="https://user-images.githubusercontent.com/65351722/81997902-16451480-961f-11ea-92bf-c15201f0bd3f.png">

**Comment by leturbinator** [comment]
sorry the bottom cut off for the first pic
<img width="509" alt="image" src="https://user-images.githubusercontent.com/65351722/81997937-307ef280-961f-11ea-876e-7500338a61f7.png">

**Comment by leturbinator** [comment]
Unfortunately there is still a discrepancy- perhaps I did it incorrectly?

**Comment by cjdduarte** [comment]
> Unfortunately there is still a discrepancy- perhaps I did it incorrectly?

No. I really expected that. I discovered the "problem". I believe it is an Anki bug. Look:
When you click on a deck to study, the information on this screen:

![image](https://user-images.githubusercontent.com/42243532/82034553-b711ef00-9674-11ea-9e72-b1e46fd1ba66.png)
**0+16+1760=1776**

It is based on this line:

![image](https://user-images.githubusercontent.com/42243532/82034666-df99e900-9674-11ea-8167-a1f80fc3fb0d.png)

![image](https://user-images.githubusercontent.com/42243532/82034947-46b79d80-9675-11ea-911c-201a90b1b07b.png)
**0+16+1209+551=1776**

The problem is that this 551 is being added to your screen, which is an error:
![image](https://user-images.githubusercontent.com/42243532/82034553-b711ef00-9674-11ea-9e72-b1e46fd1ba66.png)
Correct would be: 0+16+(1760-551)=**0+16+1209**

![image](https://user-images.githubusercontent.com/42243532/82035757-5be0fc00-9676-11ea-94dd-fdadce5f021b.png)

![image](https://user-images.githubusercontent.com/42243532/82036022-bed29300-9676-11ea-855c-e7c438ba12ad.png)
**150+19+1494=1.663**

In short, Anki is computing the value of 551 that you will not study.


![image](https://user-images.githubusercontent.com/42243532/82036674-9303dd00-9677-11ea-84a9-c5a0300fe757.png)

do a test, change its value in the review tab to 9999 (**ALL your groups**).
You are limiting the number of revisions per day.

After making this change, see if the values ​​will match.

**You will see that when you do this, the value of 551 will disappear from the screen (in fact it will be added to the amount due).**

**Comment by leturbinator** [comment]
Unfortunately my setting were already at reviews 9999. However, I tried to do what you instructed again:

**Comment by leturbinator** [comment]
<img width="423" alt="Screen Shot 2020-05-15 at 10 51 24 AM" src="https://user-images.githubusercontent.com/65351722/82064009-3960da00-969a-11ea-88f7-8a66b610096a.png">

**Comment by leturbinator** [comment]
<img width="1252" alt="Screen Shot 2020-05-15 at 10 50 03 AM" src="https://user-images.githubusercontent.com/65351722/82064027-3e258e00-969a-11ea-9d46-4e28838869ac.png">
<img width="1409" alt="Screen Shot 2020-05-15 at 10 49 53 AM" src="https://user-images.githubusercontent.com/65351722/82064028-3ebe2480-969a-11ea-8fa3-16ca88e95ba9.png">
<img width="1553" alt="Screen Shot 2020-05-14 at 8 08 05 PM" src="https://user-images.githubusercontent.com/65351722/82064029-3ebe2480-969a-11ea-9635-4175de008832.png">

**Comment by leturbinator** [comment]
oh I apologize, please ignore the last photo

**Comment by cjdduarte** [comment]
![image](https://user-images.githubusercontent.com/42243532/82076939-f19b7c00-96b4-11ea-80bb-341990e1214d.png)

are there any subdecks within the "Anking" deck, are they all using the same group (Reviews BURY)?
each group works in isolation. Can you open your subdecks?

**Comment by leturbinator** [comment]
<img width="1417" alt="image" src="https://user-images.githubusercontent.com/65351722/82081182-e431c200-96b3-11ea-821d-486cf5a7a74b.png">

**Comment by leturbinator** [comment]
I have it set to that they all follow the same group.

**Comment by cjdduarte** [comment]
Go to the "Browse" and enter this filter:
**is:due**
Enter the number of cards returned to you.

![image](https://user-images.githubusercontent.com/42243532/82119966-06c9e680-9759-11ea-9c60-36f3868c8256.png)

**Comment by leturbinator** [comment]
<img width="857" alt="image" src="https://user-images.githubusercontent.com/65351722/82122701-1f3efe80-9763-11ea-92bf-b3c7ce1a399f.png">

**Comment by leturbinator** [comment]
<img width="532" alt="image" src="https://user-images.githubusercontent.com/65351722/82122714-30880b00-9763-11ea-8757-b965247f4cf4.png">

**Comment by leturbinator** [comment]
1700 cards due today, the add-on is saying less than that though.

**Comment by cjdduarte** [comment]
Unfortunately, in order to investigate further, only if you sent me your deck. Would it be possible?
